components/Editor/Console.tsx

```diff
@@ -20,8 +20,11 @@ const Console = () => {
   return (
     <div
       ref={container}
-      className="px-4 pb-2 leading-5 font-mono text-tiny text-gray-400 dark:text-gray-500"
+      className="px-4 py-4 leading-5 font-mono text-tiny text-gray-400 dark:text-gray-500"
     >
+      <div className="text-gray-500 dark:text-gray-400 uppercase text-[11px]">
```

Comment:
Instead of `text-[11px]` lets use `font-medium` to keep consistency with titles in the debug info section